assets/wizards/advertising/components/ad-unit-size-control/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Ad Unit Size Control.
+ *
+ * Select from a subset of sizes, or enter custom width and height.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import { Grid, SelectControl, TextControl } from '../../../../components/src';
+
+/**
+ * IAB Standard Sizes.
```

Comment:
Just for reader convenience ;) And where is that list sourced from? A link would be handy here.

```suggestion
 * Interactive Advertising Bureau's standard ad sizes.
```